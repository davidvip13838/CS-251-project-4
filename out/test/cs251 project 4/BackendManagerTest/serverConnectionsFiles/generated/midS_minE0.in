500 565
273 263 4730006.24745
268 183 8273658.1357
463 429 121945.43903
152 93 2024209.53535
35 234 595658.91614
388 251 4855003.49676
148 459 7055778.69892
177 371 6698717.68712
17 235 8188840.35222
35 65 3398461.49536
421 281 1798675.8216
171 468 5908581.16164
494 224 3656938.79582
35 316 2235008.36053
493 62 5805917.14786
104 326 1394380.28061
35 411 2385884.42602
123 381 6069303.21235
35 360 4990504.58862
101 162 9042222.42707
175 17 7652011.26892
492 437 6630339.00474
13 150 3754625.18967
112 363 2965644.7675
445 305 2007363.59278
219 59 1904680.60561
53 321 4892718.30278
453 408 1919782.37591
333 317 7075329.25048
381 410 2725108.18545
293 419 4100530.93995
386 48 4079876.41152
389 5 6183347.76992
396 64 7351457.27624
235 40 6852201.90446
133 102 4782128.38449
183 247 7215157.98021
35 267 6867316.36097
22 142 3228259.57661
283 327 5551226.75568
493 381 5135353.91342
35 61 102220.38093
2 229 9360956.22991
159 29 1665694.07971
479 359 6995895.20986
32 339 1380154.78327
74 262 5090459.07437
488 373 7917141.54779
494 25 1223844.11184
272 341 1708031.77641
10 244 1943764.22667
464 405 8325226.11742
398 148 3077895.40484
494 436 5543775.2066
183 312 8626756.54203
444 134 8796282.17767
35 196 5564691.19227
254 309 9379544.43443
310 44 7492775.21221
35 85 1918935.89384
475 133 4427512.38877
451 442 8259129.06306
35 187 7095969.51782
119 313 83505.82822
8 366 1061000.77291
265 107 6081590.12968
35 380 1633234.80558
373 427 8874783.7064
185 137 3332075.63181
367 113 9116048.09819
36 317 7834968.0431
35 322 1806116.77069
311 177 2951144.18266
62 131 8817711.11754
321 27 5723897.06735
286 189 4780578.68149
35 424 2346476.85623
428 305 3645213.25685
349 24 4239617.36376
278 28 5624652.70096
35 90 972370.27489
378 198 6328974.66596
134 314 4031367.0527
195 229 4735994.12236
359 99 2630868.47694
383 486 2042103.16233
87 497 4948465.60225
95 472 1441174.11715
322 443 5167395.19668
296 101 9331452.58265
45 33 3912566.84423
208 112 514840.02196
384 96 3762474.58559
320 271 9981931.96982
292 328 7220199.64977
458 80 8851490.57308
35 23 7560437.9455
361 202 1739634.08204
35 242 6715349.05783
183 270 6598342.9594
355 263 6898842.33638
283 34 7405811.88602
310 417 9731865.16707
260 348 1391238.53342
478 195 230076.60465
35 439 9594977.11267
338 123 6082138.17322
475 147 3469645.77698
427 444 7938209.48645
129 232 2680925.16964
35 169 8547729.43336
134 343 5365400.47735
352 35 7404607.1049
35 260 9005577.01649
331 330 3844800.45138
74 478 6948108.43704
35 54 2743533.53961
329 130 1270551.30205
174 185 3169801.5801
288 391 9160334.51619
71 132 9138013.66883
397 263 87298.71701
35 457 7834633.92404
398 224 7950428.54705
9 17 2203301.15001
161 39 1459294.63217
35 368 9468848.61382
83 330 3453521.97036
341 204 5165575.20869
270 200 2541788.482
334 421 9506383.08746
231 417 5839967.99335
213 386 8749226.33618
150 101 6335111.36497
283 81 255103.56379
199 42 2416099.30248
396 278 5244819.32648
394 491 4300212.73863
212 472 6055463.39595
246 209 5949688.48098
329 40 9350669.82875
277 328 4647374.13094
84 256 37731.94457
334 283 2019461.80905
222 209 7304593.45571
283 383 3898935.3702
340 217 9666320.47275
11 296 578409.9953
154 448 295150.59564
269 253 9747372.68068
230 77 6594710.42915
449 124 3802522.69031
328 78 646628.909
116 108 188089.09814
153 30 431457.21428
35 300 6657302.286
434 450 6282269.57065
488 64 2794912.9335
286 322 9892254.9815
340 139 7829712.84358
70 463 1367237.31647
15 492 2485966.38443
351 168 94376.61852
113 265 8850951.25785
451 333 838165.52752
306 477 4341920.08642
272 213 6179089.10337
465 409 746547.90139
17 375 3175852.77149
3 327 7057488.1429
413 452 2649832.44629
484 308 8707494.26742
59 118 4987810.93033
39 385 4293578.7739
337 395 397799.30153
34 193 3214870.37848
404 170 4890225.41673
320 351 2704329.18833
460 418 438780.5748
159 459 6750888.70438
63 72 1544058.51225
206 304 5678482.58212
414 416 7430194.8451
35 257 2561876.45123
35 136 1503521.18204
71 371 336964.30995
272 83 65841.23384
110 482 8593738.27133
295 8 5196544.66198
351 191 525643.85769
378 251 7675986.52477
488 182 3477560.88526
274 462 7708210.32829
467 228 178494.1525
9 329 8283838.02319
126 60 8425348.9953
433 131 4871525.56756
102 279 4829429.5137
149 114 492606.08788
249 28 5629461.36504
203 333 2908404.67456
285 370 8434083.8643
62 384 8930478.61946
58 60 5210377.99162
35 415 7714744.22995
41 304 7211486.76346
491 378 6984633.7209
357 64 2570931.06207
11 206 7631960.70219
93 4 5916678.75158
218 371 7142397.0141
275 309 7383500.41124
35 364 307262.50549
35 151 48295.16486
230 149 7166591.99898
49 212 6169256.40838
315 103 5342538.51899
465 310 8210658.55512
209 5 7870626.49043
73 126 977792.13313
36 92 2524265.03101
134 358 3645528.65225
279 273 5519207.31038
236 197 2522226.05323
330 347 5341837.25263
117 42 9113325.4369
438 203 8801565.77928
35 496 4026993.28711
466 395 2609250.99409
15 399 6791188.36339
175 26 2618895.37111
33 130 9255300.68605
263 323 4302999.41933
479 265 3429967.37591
275 401 1176764.06315
12 25 8111008.94444
489 407 4308200.79368
35 115 7488895.49442
168 400 6278995.45015
182 127 53959.73079
459 365 5273198.06121
146 343 4926779.1922
11 69 2208060.06282
405 55 9634990.62144
412 399 9765204.94884
358 374 6825585.98814
410 304 2804004.16919
186 70 2591813.01412
361 153 8013444.91398
57 156 8135898.98838
13 33 8390999.24621
482 220 1507643.17183
386 297 6306607.91237
425 335 3785667.23835
84 1 4297257.03681
396 133 963426.83232
261 441 5741128.58945
240 449 4940811.63247
63 440 6599851.79952
238 41 6774816.79089
21 181 7277218.37117
262 456 9968506.17573
393 277 5622087.65395
284 42 6837864.80232
35 94 1006914.24311
100 16 1720976.45751
297 56 2651537.65734
446 274 1297554.26534
208 56 4973495.5927
216 319 8540647.89113
468 219 2071915.83314
492 355 4722385.84566
35 47 7386485.27064
33 35 6029067.38895
96 207 8635828.47081
35 138 4377299.45528
498 356 7801774.16091
261 408 4870220.36294
297 73 2428545.50163
297 304 6640614.54891
176 224 9177438.38659
403 236 1915129.43399
483 435 7294327.9875
29 302 3257517.70863
195 7 5358128.74495
137 440 8296510.12007
278 426 6554334.71653
35 144 8532524.59351
468 12 5427256.04887
253 488 2597534.13463
28 106 2561389.95264
284 156 8214563.36955
285 10 7835859.80974
404 478 9055366.1432
240 343 1223277.3501
499 485 3008919.22392
207 353 5602098.71758
35 97 3928936.78505
149 420 5317755.29934
425 235 6466010.34313
431 230 2157496.42182
199 182 2179593.38647
50 21 7575958.35642
478 199 3135213.7825
0 35 819558.82218
320 33 5911079.75719
202 132 8997555.14965
35 173 2647626.81568
133 103 9984737.93319
272 386 8226025.5932
244 125 6655752.52095
262 372 2428350.18406
35 264 9736961.76596
379 156 207443.36173
499 478 7916049.05819
35 461 1792960.13496
42 377 447095.47655
377 244 6749160.90955
313 202 3228879.15766
205 289 1700931.26312
35 76 9991211.67265
227 451 7092485.77236
35 299 8675952.33966
265 349 8146889.4399
35 82 7448525.3191
319 311 8722071.10655
1 78 3891029.15257
205 486 5760697.06658
298 430 5507808.37201
1 445 8113949.56748
17 78 2705930.60284
250 236 7011815.23051
329 332 5109304.17651
359 386 3706164.14699
35 228 4105801.47486
35 111 9027291.24436
215 272 2053425.61406
226 490 2383742.97974
134 291 8749316.98011
404 339 5566264.96145
35 91 9652560.0518
314 178 9388814.67945
212 346 9508466.57469
51 220 2220693.77635
219 468 5016927.33215
72 119 5271509.03009
334 453 2269624.1834
358 372 7659086.87803
116 60 9033724.82016
270 180 8228.12105
35 14 2426359.21444
68 32 674941.30472
184 455 7380965.32569
46 495 5744341.32639
213 400 9454746.83784
46 372 9153444.46552
75 175 5192052.22664
384 290 2290925.64299
35 430 2443990.41612
172 328 3625654.68469
7 212 2719800.66614
442 88 4268712.50417
200 342 6881112.41026
131 247 1033106.50671
458 376 9284899.18305
214 466 4777719.14393
356 406 5269467.61321
35 166 8019054.27137
221 356 1170003.83989
381 79 976722.50208
150 482 729238.58144
35 480 9020726.46439
470 48 7075409.9359
378 489 6515353.63017
155 333 9625966.265
235 110 7506162.51017
479 377 7510571.68607
35 369 702500.28782
35 248 3456304.15272
2 422 2063074.6685
466 431 8058804.3835
303 53 3638636.05942
484 142 7918031.76366
81 291 799683.50229
288 191 8724248.07941
428 225 3625613.27866
123 89 4549635.00583
158 456 3588301.95612
121 492 5692776.98833
250 420 7741900.08341
445 164 676847.88179
345 233 7460907.30996
265 250 97298.87745
332 141 9031132.73527
160 334 947544.6032
36 466 9890697.24308
18 347 8824820.63793
35 469 4436444.61455
28 244 2977235.36933
13 31 2945233.56731
43 83 6295903.27158
35 301 3510353.88362
202 326 3872151.51902
96 223 1690214.58704
220 237 595156.66085
88 318 4291033.52315
387 418 87443.35728
476 292 1904095.52396
312 211 9850853.6514
150 179 3701978.44251
476 219 4130677.34873
35 474 9685943.12102
381 314 9944541.95695
476 376 2004367.92642
386 233 9232226.26594
110 210 3162473.35748
156 361 6873069.80039
306 155 8339853.09875
497 322 2972649.39726
19 335 8026522.84129
60 314 6185187.17906
355 231 9063876.64787
122 230 3719563.74687
394 67 9553968.15525
222 492 6734554.10161
333 354 166892.78061
438 408 3481539.61622
145 476 9193209.15253
35 163 5190968.81858
287 386 8228151.1629
280 12 7927540.81446
99 240 6391710.5242
20 378 3431182.51244
35 239 5745278.241
324 233 3976374.15124
347 207 4055204.0249
420 15 1233058.34526
35 245 8868886.61918
35 336 7328694.7118
355 433 9514989.90263
417 88 8456013.98719
358 402 5428042.63849
390 391 3180482.75862
259 460 9843674.09129
358 127 2807652.58118
410 190 268934.86972
487 5 546050.12414
252 294 9817676.13774
420 167 1986922.10292
421 330 494332.75209
208 255 6444480.37656
273 93 1264553.62004
96 459 1592773.92035
181 36 6332362.92773
243 140 2727099.17937
292 98 8751318.51249
232 155 7013786.88037
362 357 5871262.27172
206 402 2619393.73215
188 169 6147282.16202
35 447 371235.95073
382 287 197377.18095
252 327 2633414.23658
437 56 5732838.01354
242 482 9562129.25239
69 295 3071594.35147
266 236 2465676.67505
35 157 8889585.82258
110 372 1105832.06418
48 179 5545522.31187
408 117 3041830.2221
277 79 9563968.82701
381 168 3241162.89379
158 174 2104215.84973
16 361 342386.59029
35 350 3380485.74988
9 106 4467973.98686
432 405 5022105.60403
282 429 9664503.58891
355 121 3298672.82835
35 86 2742963.17933
177 423 5109617.33346
318 419 6075261.96946
34 296 7560688.82958
280 123 5279135.075
450 446 7172650.34716
238 109 4503661.93551
35 309 7062965.16797
258 119 4430363.24286
182 420 3170383.55882
410 414 781843.62377
18 156 9824442.38282
208 237 7249460.55202
392 57 8807999.46449
244 131 1441473.6911
371 282 6801370.87679
69 340 2698129.27101
192 260 3433836.20487
313 215 7458404.10512
483 280 7745376.54235
208 454 8403973.66907
344 147 3009178.91654
105 405 4255387.41046
441 101 6124978.57784
250 325 5508318.86765
237 382 2377678.74275
407 164 9113087.44352
17 307 9975947.56565
109 205 8648503.67109
8 385 5599661.25071
211 356 9003192.77139
64 423 2649511.43627
431 8 614019.83031
226 249 3805065.97102
355 48 253815.11679
455 374 1700993.99767
365 32 1892283.4154
87 165 5330253.40267
60 459 533647.24508
242 418 4241442.01921
232 150 773193.2905
143 126 2510595.60972
106 38 7282013.02887
363 108 8844047.59183
355 238 451344.62318
209 37 497932.09129
328 398 6409695.98257
120 92 2284173.42042
290 446 9925542.01505
386 139 3064421.07849
214 367 8442789.41381
195 328 6343767.13321
402 230 9369272.24999
413 199 7503776.97596
476 171 5246305.30802
417 464 7682283.78896
481 219 200021.24442
486 352 4631009.67688
35 194 6775766.27098
49 135 468065.6654
106 313 3440505.35445
66 310 213315.38346
283 241 3118903.16123
441 392 6212777.63401
403 321 9886782.71689
247 71 2910693.53762
397 448 347497.0752
243 331 1975172.59468
305 201 2015521.87755
289 119 647174.86853
279 427 5920214.3267
142 167 7912881.83998
71 7 5444996.86413
35 276 4506895.50885
84 29 9120690.66489
201 235 788879.1054
35 6 2906387.91304
471 173 7662638.50597
52 155 370937.58458
35 473 5305153.23303
128 108 5225880.90765
426 385 9282054.52612
367 297 8316644.31709
448 99 4405656.73189
498 202 9336903.0764
