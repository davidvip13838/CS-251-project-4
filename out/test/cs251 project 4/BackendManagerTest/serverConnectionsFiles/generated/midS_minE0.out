2.28595424818267E9